app/models/user.rb

```diff
@@ -1,7 +1,10 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'user_id'
```

Comment:
- The same issue applies here, let's maintain the database structure as from the ERD diagram as this should be author_id. please, let's be consistent following the project requirement.